### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by apruneda**
Changes:
- Did an overall edit of the README and CONTRIBUTING pages
- Updated URLs
- Updated branding (Splunk Enterprise SDK)
- Fixed formatting issues

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,42 +2,34 @@
...
+If you're seeing some unexpected behavior with this project, please create an [issue](https://github.com/splunk/splunk-sdk-python/issues) on GitHub with the following information:
 
 1. Version of this project you're using (ex: 1.5.0)
-1. Platform version (ex: Windows Server 2012 R2)
-1. Framework version (ex: Python 2.7.9)
-1. Splunk version (ex: 6.3.0)
-1. Other relevant information (ex: local/remote environment, Splunk network configuration)
+2. Platform version (ex: Windows Server 2012 R2)
+3. Framework version (ex: Python 2.7.9)
+4. Splunk version (ex: 6.3.0)
```
Comment (tdhellmann) [inline CONTRIBUTING.md:14]: Can we update this to "Splunk Enterprise version (ex: 8.0)"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,42 +2,34 @@
...
-If you're seeing some unexpected behavior with this project, please create an [issue on GitHub][issues] with the following information:
+If you're seeing some unexpected behavior with this project, please create an [issue](https://github.com/splunk/splunk-sdk-python/issues) on GitHub with the following information:
 
 1. Version of this project you're using (ex: 1.5.0)
-1. Platform version (ex: Windows Server 2012 R2)
-1. Framework version (ex: Python 2.7.9)
-1. Splunk version (ex: 6.3.0)
-1. Other relevant information (ex: local/remote environment, Splunk network configuration)
+2. Platform version (ex: Windows Server 2012 R2)
+3. Framework version (ex: Python 2.7.9)
```
Comment (tdhellmann) [inline CONTRIBUTING.md:13]: Let's update this to "Framework version (ex: Python 3.7)"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,42 +2,34 @@
...
+4. Splunk version (ex: 6.3.0)
+5. Other relevant information (ex: local/remote environment, Splunk network configuration)
```
Comment (tdhellmann) [inline CONTRIBUTING.md:15]: ...standalone deployment or distributed, is a load balancer being used)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,42 +2,34 @@
...
 We love to see pull requests!
 
 To create a pull request:
 
-1. Fill out the [Individual Contributor Agreement][indivcontrib].
-1. Fork [the repository][repo].
-1. Make changes to the **`develop`** branch, preferably with tests.
-1. Create a [pull request][pulls] against the **`develop`** branch.
+1. Fill out the [Individual Contributor Agreement](https://www.splunk.com/en_us/form/contributions.html).
+1. Fork the [repository](https://github.com/splunk/splunk-sdk-python).
```
Comment (tdhellmann) [inline CONTRIBUTING.md:26]: The numbering here is messed up again as well

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,42 +2,34 @@
...
+1. Make changes to the **develop** branch, preferably with tests.
+1. Create a [pull request](https://github.com/splunk/splunk-sdk-python/pulls) against the **develop** branch.
 
 ## Contact us
 
-You can [contact support][contact] if you have Splunk related questions.
+Contact [Support](https://www.splunk.com/en_us/support-and-services.html) if you have questions.
```
Comment (tdhellmann) [inline CONTRIBUTING.md:32]: "If you have a paid Splunk Enterprise or Splunk Cloud license, you can contact..."

File: README.md [hunk 6]
```diff
@@ -1,102 +1,77 @@
...
 
-The information in this Readme provides steps to get going quickly, but for more
-in-depth information be sure to visit the
-[Splunk Developer Portal](http://dev.splunk.com/view/SP-CAAAEBB).
 ### Requirements
 
-Here's what you need to get going with the Splunk SDK for Python.
+Here's what you need to get going with the Splunk Enterprise SDK for Python.
+
+* Python 2.7+ and Python 3.
```
Comment (tdhellmann) [inline README.md:27]: "Python 2.7 or Python 3.7"

File: README.md [hunk 7]
```diff
@@ -1,102 +1,77 @@
...
-in-depth information be sure to visit the
-[Splunk Developer Portal](http://dev.splunk.com/view/SP-CAAAEBB).
 ### Requirements
 
-Here's what you need to get going with the Splunk SDK for Python.
+Here's what you need to get going with the Splunk Enterprise SDK for Python.
+
+* Python 2.7+ and Python 3. 
+  
+  The Splunk Enterprise SDK for Python has been tested with Python v2.7 and v3.5.
```
Comment (tdhellmann) [inline README.md:29]: v3.7

File: CONTRIBUTING.md (hunk 4 above)
Comment (apruneda) [reply CONTRIBUTING.md]: fixed

**Review by tdhellmann — APPROVED** [review]

**Pull request merged**